rebar.config

```diff
@@ -20,234 +20,271 @@
-        %% off-the-shelf depdendencies
-
-        {enoise, {git, "https://github.com/aeternity/enoise.git",
-                 {ref, "1e6ee67"}}},
-
-        {gproc, "0.6.1"},
-        {jobs, "0.9.0"},
-        {exometer_core, "1.5.7"},
-        {yamerl, "0.7.0"},
-        {eper, "0.99.1"},
-        {ephemeral, "2.0.4"},
-        {sext, "1.8.0"},
-
-        {lager, {git, "https://github.com/aeternity/lager.git",
-                {ref, "9d97ae3"}}}, % with trace-based msg suppression
-        {cowboy, {git, "https://github.com/ninenines/cowboy.git",
-                 {ref, "8d49ae3"}}}, % tag: 2.2.2"
-        {idna, {git, "https://github.com/benoitc/erlang-idna",
-               {ref, "6cff727"}}}, % tag: 6.0.0
-        {nat, {git, "https://github.com/aeternity/erlang-nat.git",
-              {ref, "dcdfb9c"}}},
-        {jsx, {git, "https://github.com/talentdeficit/jsx.git",
-              {ref, "3074d48"}}},
-        {lz4, {git, "https://github.com/szktty/erlang-lz4.git",
-              {ref, "9ea04f2"}}},
-
-        %% deps originally from aeternity
-
-        % The rocksdb dependencies are removed on win32 to reduce build times,
-        % because they are currently not working on win32.
-        {mnesia_rocksdb, {git, "https://github.com/aeternity/mnesia_rocksdb.git",
-                         {ref, "66e5a51"}}},
-
-        {mnesia_leveled, {git, "https://github.com/aeternity/mnesia_leveled.git",
-                         {ref, "11c480b"}}},
-
-        {aeminer, {git, "https://github.com/aeternity/aeminer.git",
-                  {ref, "0a82f0f"}}},
-
-        {aebytecode, {git, "https://github.com/aeternity/aebytecode.git",
-                     {ref, "05dfd7f"}}},
-
-        {aeserialization, {git, "https://github.com/aeternity/aeserialization.git",
-                          {ref,"58e34ae"}}},
-
-        {aestratum_lib, {git, "https://github.com/aeternity/aestratum_lib.git",
-                        {ref, "a043272"}}},
-
-        {ecrecover, {git, "https://github.com/aeternity/ecrecover.git",
-                    {ref, "b3b6421"}}},
-
-        {emcl, {git, "https://github.com/aeternity/emcl.git",
-               {ref, "76a493b"}}},
-
-        %% forks
-
-        {enacl, {git, "https://github.com/aeternity/enacl.git",
-                {ref, "a45b433"}}},
-
-
-        {jesse, {git, "https://github.com/for-GET/jesse.git",
-                {ref, "bf5ba4b"}}},
-
-        % upstream is not maintained anymore
-        {base58, {git, "https://github.com/aeternity/erl-base58.git",
-                 {ref,"60a3356"}}},
-
-        % upstream is not maintained anymore
-        {sha3, {git, "https://github.com/aeternity/erlang-sha3",
-               {ref, "b5f27a2"}}},
-
-        % for lightweight trace-based debugging
-        {trace_runner, {git, "git://github.com/uwiger/trace_runner.git",
-                        {ref, "69dc388"}}},
+	%% off-the-shelf depdendencies
+	{enoise, {git, "https://github.com/aeternity/enoise.git", {ref, "1e6ee67"}}},
```

Comment:
If tidying up, this one isn't off-the shelf 😅

Reply:
I wouldn't want to cut any pre-existing comments -- there is a reason for their existence probably. Are you certain it must go?

Reply:
No, I mean the dependency should be moved further down, to a group where it belongs.